awx/ui/src/components/Search/AdvancedSearch.js

```diff
@@ -86,7 +118,12 @@ function AdvancedSearch({
       const actualSearchKey = [actualPrefix, keySelection, lookupSelection]
         .filter((val) => !!val)
         .join('__');
-      onSearch(actualSearchKey, searchValue);
+      if (keySelection === 'ansible_facts') {
+        const ansibleFactValue = [actualSearchKey, searchValue].join('__');
```

Comment:
why not use `${actualSearchKey}__${searchValue}` in a string? Seems a little more direct than using `join`